bedrock/firefox/templates/firefox/desktop/index.html

```diff
@@ -21,7 +21,7 @@
   {% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
Reason they remain site_css is because they don't need tabzilla-static css because these templates all use the familynav. However index.html should also remain as site_css otherwise it has extra css it doesn't need. I would need to add back sandstone-less file into the  firefox-desktop bundle, since thats the only css index.html needed to inherit.

For me to update the site_css to page_css for these I would need to update the base template to use familynav. But since we don't want that (I am assuming familynav was specifically added to pages) for now leaving them as site_css and making a PR for maybe down the road removing this familynav implemenation of the tabzilla tab and using tabzilla-static instead. 

The only thing I'll be fixing here is adding back sandstone-less to firefox-desktop and updating the index to site_css block since firefox-desktop would have all the needed css.

Does that make sense @jpetto?